lib/main.php

```diff
@@ -56,7 +56,7 @@ private function api_request ($endpoint, $method = 'GET', $data = null) {
     return $result;
   }
 
-  function xml ($title, $query_string, $set_properties, $product_ids, $search_endpoint, $offset = 0, $is_list_all, $output_file, $is_skip_variations = false, $discount = 0) {
+  function xml ($title, $query_string, $set_properties, $product_ids, $search_endpoint, $offset = 0, $is_list_all, $output_file, $is_skip_variations = false, $discount = 0, $utm) {
```

Comment:
```suggestion
  function xml (
    $title,
    $query_string,
    $set_properties,
    $product_ids,
    $search_endpoint,
    $offset = 0,
    $is_list_all,
    $output_file,
    $is_skip_variations = false,
    $discount = 0,
    $utm
  ) {
```

Esse trem já tá ilegível, deveria virar um array `options`